.github/workflows/deploy-pages.yml

```diff
@@ -69,6 +70,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       HUGO_VERSION: 0.117.0
+    permissions:
+      contents: write
```

Comment:
Why we need this + `git commit` below?

Reply:
@lidel (see the other comment) we deploy the `fleek-deploy` branch to fleek.
We build within Github's CI, then commit & push to the branch used by fleek.